src/Forms/book-form/BookForm.jsx

```diff
@@ -1,15 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Style from './BookForm.module.css';
 
-/* eslint-disable jsx-a11y/anchor-is-valid, react/jsx-curly-brace-presence */
-export default function BookForm() {
+/* eslint-disable jsx-a11y/anchor-is-valid, react/jsx-curly-brace-presence, react/prop-types */
```

Comment:
Kindly take away every line of code that disables `eslint` in your application😉. we are required to make use of linters to ensure we write code void of bad practices😉. Kindly do well to fix all your linter errors👍🏼.